style.css

```diff
@@ -0,0 +1,100 @@
+html {
+  padding: 24px;
+}
+
+.toolbar {
+  /* Welcome */
+  height: 28px;
+  left: 12px;
+  top: 10px;
+  padding-left: 0%;
+
+  /* Theme 2 / Subhead 1 */
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+
+  /* identical to box height, or 140% */
+  display: flex;
+  justify-content: space-between;
+
+  /* Theme 2 / Neutral 1 */
+  color: #28352f;
+
+  /* changing list style */
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  text-align: left;
+  margin-top: 114 px;
+}
+
+.headline h1 {
+  font-family: 'Poppins', sans-serif;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  text-align: left;
+  margin-bottom: 0%;
+}
+
+.headline p {
+  /* Supporting Text */
+  height: 164px;
+  left: 24px;
+  top: 314px;
+  margin: 2px;
```

Comment:
- [Optional] Hey @dariamine2000 , so what was happening was that your `<p>` had a fixed `height: 164px` as you can see above. And according to the box model, when the content of your box is moving to the next line, your `<p>` height is not increasing with it which causes the text to overflow and overlap the other elements. What you can do to fix it, is use a `min-height: 164px` instead which will let it have a minimum height of 164px and no less, but also let it increase when the width is getting smaller and the height has to increase to contain the content 🌥️